Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -283,6 +283,28 @@ public void ProcArgs_HostUrl_SonarcloudUrl_PropertyAggregation()
                 "Please set either 'sonar.host.url' for SonarQube or 'sonar.scanner.sonarcloudUrl' for SonarCloud.");
             sut.IsValid.Should().BeFalse();
         }
+
+        [DataTestMethod]
+        [DataRow(true, true, true, 3)]
+        [DataRow(false, true, true, 2)]
+        [DataRow(true, false, true, 2)]
+        [DataRow(true, true, false, 2)]
+        [DataRow(false, false, true, 1)]
+        [DataRow(false, true, false, 1)]
+        [DataRow(true, false, false, 1)]
+        [DataRow(false, false, false, 0)]
+        public void ProcArgs_ErrorAndIsValid(bool invalidKey, bool invalidOrganization, bool invalidHost, int errors)
```

Comment:
This test was a leftover from #1984. The PR was merged before I could push the test.